editor/editor_visual_profiler.cpp

```diff
@@ -667,6 +667,7 @@ void EditorVisualProfiler::_combo_changed(int) {
 void EditorVisualProfiler::_bind_methods() {
 
 	ADD_SIGNAL(MethodInfo("enable_profiling", PropertyInfo(Variant::BOOL, "enable")));
+	ADD_SIGNAL(MethodInfo("break_request"));
```

Comment:
This has been removed several times, it should not be readded :)

It's only relevant for the script profiler, not for the visual and network profilers.

Reply:
It was removed in a4125bb, maybe it's not in this PR branch. I'll rebase.

Reply:
So we need to delete this connect?
```
visual_profiler->connect("break_request", callable_mp(this, &ScriptEditorDebugger::_profiler_seeked));
```

Reply:
It was already deleted in a4125bbe, I just rebased to include it. Now you'll have to rebase too.